flow/util/test/test_genElapsedTime.py

```diff
@@ -28,23 +30,24 @@ def test_elapsed_time(self, mock_stdout):
             f.write("Some log entry\n")
             f.write("Elapsed time: 01:30:00[h:]min:sec.\n")
         # call the script with the test log file
-        sys.argv = ["util/genElapsedTime.py", "--logDir", str(self.tmp_dir.name)]
+        sys.argv = ["./genElapsedTime.py", "--logDir", str(self.tmp_dir.name)]
         with patch.object(sys, 'argv', sys.argv):
-                import genElapsedTime
+            module = importlib.import_module(self.module_name)
```

Comment:
Is this missing the `importlib.reload(module)` like on the other tests?

Reply:
I need to reload for the rest of the test cases but the first test case I only need importing